### PR TITLE
HAWQ-1323. Add license header added to each source file in test directory

### PR DESCRIPTION
**Pull request by amyrazz44**

### CODE REVIEW

**Comment by amyrazz44** [comment]
@huor @paul-guo- @linwen please help review this pull request. thank you

**Comment by huor** [comment]
+1

File: pom.xml [hunk 1]
```diff
@@ -129,7 +129,42 @@
...
+              <exclude>src/test/performance/sqls/drpsimple</exclude>
+              <exclude>src/test/performance/sqls/orbsimple</exclude>
+              <exclude>src/test/performance/sqls/slcsimple</exclude>
+              <exclude>src/test/**/Makefile</exclude>
+              <exclude>src/test/performance/results/PgSQL.970926</exclude>
+              <exclude>src/test/performance/sqls/connection</exclude>
+              <exclude>src/test/feature/doxygen_template</exclude>
+              <exclude>src/test/regress/expected_statuses</exclude>
+              <exclude>src/test/feature/lib/global_init_file</exclude>
+              <exclude>src/test/feature/ao/sql/init_file</exclude>
```
Comment (paul-guo-) [inline pom.xml]: replace all init_file lines with one line?
<exclude>src/test/feature/**/init_file</exclude> ?

**Comment by paul-guo-** [comment]
+1

**Pull request merged**